src/components/transactions/transactions.component.js

```diff
@@ -39,6 +41,15 @@ const TransactionsConstructor = function ($rootScope, $stateParams, $state, $htt
 		});
 	};
 
+	vm.makePages = (page) => {
+		const arr = [];
+		const n = Number(page);
+		if (n === 0 || n === 1 || n === 2) {
+			arr.push(1, 2, 3, 4, 5);
+		} else { arr.push(n - 2, n - 1, n, n + 1, n + 2); }
+		return arr;
+	};
```

Comment:
Think about the last page, ie. [36, 37, 38, 39, 40]. 40 is the last page available and should be highlighted.